### PR TITLE
fix: handle core flags for webpack 4

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
fix

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
https://github.com/webpack/webpack-cli/pull/2009#discussion_r515927500

**Does this PR introduce a breaking change?**
Nope

**Other information**
/cc @evilebottnawi

### CODE REVIEW

File: packages/webpack-cli/lib/webpack-cli.js [hunk 1]
```diff
@@ -39,8 +41,8 @@ class WebpackCLI {
                 acc[toKebabCase(cur)] = parsedArgs[cur];
                 return acc;
             }, {});
-        const coreCliHelper = require('webpack').cli;
         const coreCliArgs = coreCliHelper.getArguments();
+        console.log(coreCliArgs, coreCliHelper);
```
Comment (snitin315) [inline packages/webpack-cli/lib/webpack-cli.js:45]: 👁️ 👁️

Comment (anshumanv) [reply packages/webpack-cli/lib/webpack-cli.js:45]: big oops! 😂

Comment (anshumanv) [inline packages/webpack-cli/lib/webpack-cli.js:45]:
```suggestion
```

**Review by evilebottnawi — APPROVED** [review]
/cc @webpack/cli-team

**Review by snitin315 — APPROVED** [review]

**Pull request merged**